--- cmake/finds/FindNANOFLANN.cmake	2019-07-23 18:25:38.576773281 -0400
+++ cmake/finds/FindNANOFLANN.cmake	2019-07-23 18:46:24.198508386 -0400
@@ -10,9 +10,8 @@
 
 else()
 
-	find_path(NANOFLANN_INCLUDE_DIR 
-	          HINTS ${MUQ_NANOFLANN_DIR}
-			  PATH_SUFFIXES nanoflann NO_DEFAULT_PATH)
+	find_path(NANOFLANN_INCLUDE_DIR nanoflann.hpp 
+	          HINTS ${MUQ_NANOFLANN_DIR}/include NO_DEFAULT_PATH)
 					
 endif()
 set(NANOFLANN_INCLUDE_DIRS ${NANOFLANN_INCLUDE_DIR} )
