--- cmake/SetupCompiler.cmake	2019-07-23 18:25:38.576773281 -0400
+++ cmake/SetupCompiler.cmake	2019-07-23 19:36:38.928149438 -0400
@@ -23,7 +23,7 @@
 set(CMAKE_CXX_FLAGS_DEBUG  "-O0") #-O0 works better for memcheck
 set(CMAKE_CXX_FLAGS_RELEASE  "-O3") #full optimization with debug symbols for profiling
 
-set(CMAKE_CXX_FLAGS "-g")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
 
 # default to a release build
 message(STATUS "User defined build type = " ${CMAKE_BUILD_TYPE})
